### DISCUSSION TITLE
Question: Why ssh: root@168.192.1.168: Permission denied . can't change to another user to run bash script?

### DISCUSSION THREAD

**Discussion by yasok787** [semaphoreui/semaphore#2119] [Q&A]
### Question

#Semaphore UI ansible error run bash scripts 
9:43:31 PM
Task 86 added to queue
9:43:32 PM
Started: 86
9:43:32 PM
Run TaskRunner with template: repmgr
9:43:32 PM
Preparing: 86
9:43:33 PM
test-db | UNREACHABLE! => {
9:43:33 PM
    "changed": false,
9:43:33 PM
    "msg": "Failed to connect to the host via ssh: root@168.192.1.168: Permission denied (publickey,gssapi-keyex,gssapi-with-mic,password).",
9:43:33 PM
    "unreachable": true
9:43:33 PM
}
9:43:33 PM
Running playbook failed: exit status 4

**Comment by fiftin** [comment]
Hi @yasok787 please provide more details.
